frontend/packages/client/src/components/Proposal/VoteOptions/PostVoteOptions.js

```diff
@@ -0,0 +1,53 @@
+import { useState } from 'react';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+import { useModalContext } from 'contexts/NotificationModal';
+import { Svg } from '@cast/shared-components';
+import NotificationsModal from 'components/modals/Notifications';
+
+const PostVoteOptions = ({ communityId }) => {
+  const [linkCopied, setLinkCopied] = useState(false);
+  const { openModal, closeModal } = useModalContext();
+  const openNotificationsModal = () =>
+    openModal(
+      <NotificationsModal onClose={closeModal} communityId={communityId} />,
+      {
+        classNameModalContent: 'modal-content-sm',
+      }
+    );
+  return (
+    <div className="">
+      <div className="is-flex is-align-items-center">
+        <Svg name="CheckMark" />
+        <b className="is-size-4 ml-2">
+          You successfully voted on this proposal!
+        </b>
+      </div>
+      <div className="is-flex is-align-items-center mt-5">
+        <div className="is-flex flex-1 is-align-items-center rounded-lg button twitter-button mr-2">
+          <Svg name="Twitter" />
+          <span className="ml-1">Share on Twitter</span>
+        </div>
```

Comment:
I think you'll need this code https://github.com/DapperCollectives/CAST/blob/bfbb15863b75dfcb4052fb44ceaabba515144c8d/frontend/packages/client/src/components/Proposal/ShareDropdown.js#L70 to allow sharing

Reply:
updated https://github.com/DapperCollectives/CAST/pull/633/commits/748d6e30258766620742e3df2463506c9dae0698